test/integration/things-media.js

```diff
@@ -0,0 +1,201 @@
+const CONFIG = require('config')
+const __ = CONFIG.universalPath
+const _ = __.require('lib', 'utils')
+
+require('should')
+
+const breq = require('bluereq')
+const get = (url) => breq.get(url).then(_.property('body'))
+const post = (url, body) => breq.post(url, body).then(_.property('body'))
+const put = (url, body) => breq.put(url, body).then(_.property('body'))
+const delete_ = (url) => breq.delete(url).then(_.property('body'))
+
+const endpoint = CONFIG.server.url() + '/media'
```

Comment:
This can happen decoupled from the storage of media metadata. It would be up to the user client to update the respective `/place/UUID` document with an array of UUIDs of associated media.